src/library/scala/concurrent/Future.scala

```diff
@@ -618,7 +664,7 @@ object Future {
    */
   def sequence[A, M[X] <: TraversableOnce[X]](in: M[Future[A]])(implicit cbf: CanBuildFrom[M[Future[A]], A, M[A]], executor: ExecutionContext): Future[M[A]] = {
     in.foldLeft(successful(cbf(in))) {
-      (fr, fa) => for (r <- fr; a <- fa) yield (r += a)
```

Comment:
>Just trying to deconstruct this change, I'm always a bit wary of the "... and just changed a little detail in the implementation" parts of doc commits

Now that, @retronym, is a sign of good health :)

You're right on those accounts, and also: it shows what `zipWith` is good for (the code which was replaced is an alternate, slower encoding).

(Also, I'd argue, that it makes the code easier to reason about since you don't have to desugar the FC in your head :))